whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -729,6 +729,35 @@ public void lookup_object_text_plain_not_found_extension() {
         }
     }
 
+    @Test
+    public void lookup_object_text_plain_bad_source_accept_header() {
+        try {
+            RestTest.target(getPort(), "whois/oez/org/MKQ-RIPE.txt")
+                    .request()
+                    .get(String.class);
+            fail();
+        } catch (BadRequestException e) {
+            final String response = e.getResponse().readEntity(String.class);
+            assertThat(response, is("Severity: Error\n" +
+                    "Text: Invalid source '%s'\n" +
```

Comment:
use `ErrorMessage.toString()` rather than `getText()`

Reply:
ok, now, returning the errorMessage we have oez instead of %s